### DISCUSSION TITLE
`-F keywordcase:'upper'` not working

### DISCUSSION THREAD

**Discussion by goyalyashpal** [pygments/pygments#2554] [Q&A] [answered]
### Title/Summary

* `F`ilter keywordcase is not working 

### Details:

* i used it via cli
* i couldn't find any working examples in documentation either: [pyg-docs/filters]
* i tried on `l`exers `psql` and `pascal` : psql as sql is case insensitive, and pascal as it's mentioned in the docs above

[pyg-docs/filters]: https://pygments.org/docs/filters/#KeywordCaseFilter "filters on pygments docs"

### Examples

```shell
$ pygmentize -l psql -F keywordcase:'upper' <<< "db-# select count(*) as tota
l from std;"
db-# select count(*) as total from std;

```

pascal example code took from: [tpt-pascal/hworld]

[tpt-pascal/hworld]: https://www.tutorialspoint.com/pascal/pascal_program_structure.htm "pascal hello world"

```shell
$ pygmentize -l pascal -F keywordcase:upper <<< "program HelloWorld;
uses crt;

(* Here the main program block starts *)
begin
   writeln('Hello, World!');
   readkey;
end. "
program HelloWorld;
uses crt;

(* Here the main program block starts *)
begin
   writeln('Hello, World!');
   readkey;
end. 

```

**Comment by jeanas** [comment, marked as answer]
Usage is not `pygmentize -F keywordcase:'upper'` but `pygmentize -F keywordcase:case=upper`.

**Reply by goyalyashpal** [reply]
that's sooo embarrassing 😅 
sorry to bother
